cookbook/deployment/cluster/auth_setup.py

```diff
@@ -328,179 +257,79 @@
 Continuous Integration - CI
 ***************************
```

Comment:
is this meant to be under "Apply Configuration", or a sibling header  to "OAuth2 Authorization Server"?

in the latter case this should use the `==============` header style